imports/client/components/NotificationCenter.tsx

```diff
@@ -268,8 +292,12 @@ const GuessMessage = React.memo(({
             </OverlayTrigger>
           </StyledNotificationActionItem>
           <StyledGuessDetails>
-            <GuessDirection value={guess.direction} />
-            <GuessConfidence id={`notification-guess-${guess._id}-confidence`} value={guess.confidence} />
+            <GuessDirectionWrapper>
```

Comment:
Just a note: With the changes that have since landed, I don't believe these wrappers are necessary anymore.